account_events.go

```diff
@@ -62,6 +62,8 @@ const (
 	ActionCommunityQuestionReply   EventAction = "community_question_reply"
 	ActionCommunityLike            EventAction = "community_like"
 	ActionCreateCardUpdated        EventAction = "credit_card_updated"
+	ActionDatabaseCreate           EventAction = "database_create"
+	ActionDatabaseDelete           EventAction = "database_delete"
 	ActionDiskCreate               EventAction = "disk_create"
```

Comment:
Could we also add `database_update`, `database_update_failed`, `database_backup_restore`, and `database_credentials_reset`?